### DISCUSSION TITLE
V-MODEL BINDING

### DISCUSSION THREAD

**Discussion by robcisneros** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#496] [Q&A]
Hola, ¿en que momento explicaron la action en store llamada changeColor? La quiero ocupar para binding entre mis checkboxes de sabores de pastel y mutar un array que se encuentra en el store pero no logro comprender su funcionamiento ni el código de ColorCode
![duda1](https://user-images.githubusercontent.com/86903400/159628113-e2cb745e-6e56-456a-80c7-2121bcdd9e60.PNG)

**Comment by doguedogue** [comment]
https://web.microsoftstream.com/video/6fa3c8b3-a840-4cf2-998a-04007f8bc0ab?list=user&userId=96eeb66f-b984-4083-8ea6-74c34a07a34a
